### ISSUE TITLE
[MAS4.2.10][Screen reader-File->New bot configuration] In browse mode nvda is reading the at a time for Hide copy controls information

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1806]
**Actual Result:**
In browse mode NVDA is reading the at a time for Hide copy controls information
In browse mode NVDA is reading as “Hide link and Copy link “when the focus is on Hide link and focus is not moving to the copy link while press the down arrow key 

Expected Result:
In browse mode NVDA should read as hide link when the focus is on Hide link after press the down arrow key focus should moves to the copy link and nvda should read as Copy link 

**Repro Steps:**
Turn on the NVDA 
1.Open the Bot Framework V4 Emulator.
2. Navigate to the File menu and select it
3.File menu gets opened  
4.Navigate to the New bot configuration and select it 
5.New bot configuration dialog gets opened 
6.Naviage to the Hide and copy controls using Browse mode (Down arrow key)
7.Observe the issue 

**User Impact:**
Users who rely on screen reader will be affected if the screen reader reading the at a time for both controls info then the screen reader users will get confuse.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Screen reader: NVDA

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}
[MAS4.2.10_File.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562177/MAS4.2.10_File.zip)

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version 4.5.2-80829 with NVDA(2019.2) issue still repro'ing. Hence we are reactivating this bug.
Please refer the attachment:1806.zip
[1806.zip](https://github.com/microsoft/BotFramework-Emulator/files/3676274/1806.zip)

**Comment by corinagum** [comment]
Investigation notes:

- Only repros when navigating via Ctrl + DownArrow in browse mode (no repro via tab)
- Clicking anywhere in the dialog that is not interact-able when NOT in browse mode reads the text **"If you don't have a bot configuration, button create a new bot configuration".** - Note that this is the text/button that originally opened the dialog.
- When Encrypt keys checkbox is *not checked*, the following is read in browse mode:
  - Encrypt keys stored in your bot configuration. 
  - Your keys are not encrypted
  - Ctrl + DownArrow
  - Link learn more about file encryption.
  - Ctrl + DownArrow
  - Secret
  - Ctrl + DownArrow 
  - Your keys are not encrypted edit unavailable
  - Ctrl + DownArrow
  - **List with two buttons unavailable Hide secret button unavailable copy** - I'm not 100% positive this is incorrect behavior, since the text and two buttons are non-interact-able. Should non-enabled buttons behave like text?
- When Encrypt keys *is checked*, the following is read in browse mode:
  - Link learn more about bot file encryption
  - Ctrl + DownArrow
  - **Your bot is not encrypted edit unavailable - starts reading secret-** - Should not be reading "Your bot is not encrypted"
  - Ctrl + DownArrow
  - **List with two items button Hide Secret button copy.** - This definitely should be read on individual browse navigations
- Ctrl + DownArrow
  - **<Copy is not navigable via Ctrl + DownArrow/browse mode**

I have **bolded** the areas of interest for deciphering where incorrect behavior is.

**Comment by Kumar2608** [comment]
Issue got fixed hence we are clsoing this bug.
